### PR TITLE
Add CreateTransactionGroup API page

### PR DESCRIPTION
**Pull request by petitnau**
Work on DeCash-Official/decash-api#70

### CODE REVIEW

File: docs/api/algorand/create-transaction-group.mdx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+
+|        Name       |                   Type             | Description |
+| ----------------- | ---------------------------------- | ----------- |
+| `type`            | `"applicationCall"`                | Field indicating that this txnObject describes an application call. |
+| `from`            | `string`                           | The address of the account that is calling the application. |
+| `app`             | `string`                           | The ID of the application that is being called. |
+| `onComplete`      | `"NoOp"`\|`"OptIn"`\|<br/>`"CloseOut"`\|`"Update"`\|`"Delete"` | OPTIONAL. Describes the action that must be done after the call execution. By default is set to `"NoOp"`. |
+| `args`            | `Array<string`\|`number>`             | OPTIONAL. The arguments that must be passed to the application. By default is set to the empty list. |
+| `accounts`        | `Array<string>`                    | OPTIONAL. The accounts arguments. |
```
Comment (nikitaeverywhere) [inline docs/api/algorand/create-transaction-group.mdx]: I would describe it as "a list of accounts that this transaction will have access to"

Comment (nikitaeverywhere) [reply docs/api/algorand/create-transaction-group.mdx]: Same for descriptions below

File: docs/api/algorand/create-transaction-group.mdx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+| `assets`          | `Array<number>`                    | OPTIONAL. The assets arguments. |
+| `applications`    | `Array<number>`                    | OPTIONAL. The applications arguments. |
+| `approvalProgram` | `string`                           | OPTIONAL. Must be set if the `onComplete` field is set to `update`. Must be equal to the new approval program, encoded in base64.
+| `clearProgram`    | `string`                           | OPTIONAL. Must be set if the `onComplete` field is set to `update`. Must be equal to the new clear program, encoded in base64.
+
+
+|     Name    |                   Type                   | Description |
+| ----------- | ---------------------------------------- | ----------- |
+| `type`      | `"raw"`                                  | Field indicating that this is a raw txnObject. |
```
Comment (nikitaeverywhere) [inline docs/api/algorand/create-transaction-group.mdx:52]: 👍

**Review by nikitaeverywhere — APPROVED** [review]
This is so beautiful! 😍 Thank you Roberto.

File: docs/api/algorand/create-transaction-group.mdx (hunk 1 above)
Comment (petitnau) [reply docs/api/algorand/create-transaction-group.mdx]: Yep, might also be good to link some algorand docs

**Pull request merged**